packages/client-sdk-nodejs/src/cache-client.ts

```diff
@@ -79,6 +79,12 @@ export class CacheClient extends AbstractCacheClient implements ICacheClient {
 
     this.logger = configuration.getLoggerFactory().getLogger(this);
     this.logger.debug('Creating Momento CacheClient');
+
+    this._configuration.getMiddlewares().map(m => {
```

Comment:
supernit - I think in JS/TS it's idiomatic to use `forEach` instead of `map` if you are just running things for side effects, `map` collects the return values for each iteration of the loop, which you don't care about here.

Reply:
Good call, I was just copying from the `close` methods . . . I'll go ahead and convert them both.